setup.py

```diff
@@ -0,0 +1,41 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="deepquest",
+    version="0.0.1",
+    author="David Wilby",
+    author_email="d.wilby@sheffield.ac.uk",
+    description="A framework for neural-based quality estimation for machine translation",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/RSE-Sheffield/deepquest/",
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "Operating System :: OS Independent",
+    ],
+    install_requires=[
+        "future",
+        "numpy<1.17,>=1.14.5",
```

Comment:
that's slightly surprising and might need a comment to justify it

Reply:
Note to self: is this not set in the TensorFlow requirements?

Reply:
There was an incompatibility warning between tensorflow and numpy, but this has now been resolved as discussed here https://github.com/tensorflow/tensorflow/pull/30559#issuecomment-518258441
Have removed these requirements here.